buildSrc/src/main/kotlin/Dependencies.kt

```diff
@@ -70,7 +70,8 @@ object Libraries {
         const val scarlet = "0.1.12"
         const val fragment = "1.2.5"
         const val compose = "1.0.5"
-        const val composeMaterial = "1.0.0-alpha03"
+        const val composeMaterial = "1.0.5"
```

Comment:
isn't this version the same as the one above, `compose` ?

Reply:
As seen in Google Version docs, it seems `compose`and `composeMaterial` share the same release versioning, so please merge these 2
Source:
https://developer.android.com/jetpack/androidx/releases/compose-material
https://developer.android.com/jetpack/androidx/releases/compose-ui